src/forms/FormAutocomplete.jsx

```diff
@@ -106,7 +109,7 @@ export function FormAutocomplete({
       close();
       setFocus(false);
     }
-  }, [close, ignoreBlur, searchValue, setValue, updateSearchInputValidation, value]);
+  }, [close, ignoreBlur, searchValue, setValue, updateSearchInputValidation, value, onSelectItem, allowCustomInput]);
```

Comment:
@vbarcellos check the warning on this line

Reply:
Fixed